samples/nrf9160/pdn/src/main.c

```diff
@@ -44,6 +48,13 @@ void main(void)
 	uint8_t cid;
 	char apn[32];
 
+	struct nrf_ifaddrs *ifaddrs;
+	struct nrf_ifaddrs *ifaddr;
```

Comment:
At first sight this naming doesn't convince because both `ifaddr` and `ifaddrs` have the same type but satisfy different roles. How about integrating in the variable name that `ifaddr` is an iterator of sorts?

Reply:
I've changed the iterator name to `ifa` and placed it closer to the while loop, so hopefully that should make it clearer :)